Gruntfile.js

```diff
@@ -100,6 +84,65 @@ module.exports = function( grunt ) {
 				}
 			}
 		},
+		karma: {
+			options: {
+				customContextFile: "karma/context.html",
```

Comment:
I'm not 100% sure, but it seems like we don't need the custom files. With the custom things for core removed, it looks like they're just the default now. Do the tests can pass without these files?

Reply:
@Krinkle the standard file seems to run fine, but when i use the debug file I don't see any output on the screen. Also, on Windows although I can run the tests for debug I'm not sure how to exit the browser it launches in a way that Karma understands it was successful. If I close the browser it thinks Chrome crashed so it tries to relaunch 2 times and then hangs my command prompt.